contracts/0.8.25/vaults/Dashboard.sol

```diff
@@ -0,0 +1,324 @@
+        return vaultSocket().reserveRatioThreshold;
+    }
+
+    /**
+     * @notice Returns the treasury fee basis points.
+     * @return The treasury fee in basis points as a uint16.
+     */
+    function treasuryFee() external view returns (uint16) {
+        return vaultSocket().treasuryFeeBP;
+    }
+
+    // ==================== Vault Management Functions ====================
+
+    /**
+     * @notice Transfers ownership of the staking vault to a new owner.
+     * @param _newOwner Address of the new owner.
+     */
+    function transferStVaultOwnership(address _newOwner) external virtual onlyRole(DEFAULT_ADMIN_ROLE) {
+        _transferStVaultOwnership(_newOwner);
+    }
+
+    /**
+     * @notice Disconnects the staking vault from the vault hub.
+     */
+    function disconnectFromVaultHub() external payable virtual onlyRole(DEFAULT_ADMIN_ROLE) {
+        _disconnectFromVaultHub();
+    }
+
+    /**
+     * @notice Funds the staking vault with ether
+     */
+    function fund() external payable virtual onlyRole(DEFAULT_ADMIN_ROLE) {
+        _fund();
+    }
+
+    /**
+     * @notice Withdraws ether from the staking vault to a recipient
+     * @param _recipient Address of the recipient
+     * @param _ether Amount of ether to withdraw
+     */
+    function withdraw(address _recipient, uint256 _ether) external virtual onlyRole(DEFAULT_ADMIN_ROLE) {
+        _withdraw(_recipient, _ether);
+    }
+
+    /**
+     * @notice Requests the exit of a validator from the staking vault
+     * @param _validatorPublicKey Public key of the validator to exit
+     */
+    function requestValidatorExit(bytes calldata _validatorPublicKey) external onlyRole(DEFAULT_ADMIN_ROLE) {
+        _requestValidatorExit(_validatorPublicKey);
+    }
+
+    /**
+     * @notice Deposits validators to the beacon chain
+     * @param _numberOfDeposits Number of validator deposits
+     * @param _pubkeys Concatenated public keys of the validators
+     * @param _signatures Concatenated signatures of the validators
+     */
+    function depositToBeaconChain(
+        uint256 _numberOfDeposits,
+        bytes calldata _pubkeys,
+        bytes calldata _signatures
+    ) external virtual onlyRole(DEFAULT_ADMIN_ROLE) {
+        _depositToBeaconChain(_numberOfDeposits, _pubkeys, _signatures);
+    }
+
+    /**
+     * @notice Mints stETH tokens backed by the vault to a recipient.
+     * @param _recipient Address of the recipient
+     * @param _tokens Amount of tokens to mint
+     */
+    function mint(
+        address _recipient,
+        uint256 _tokens
+    ) external payable virtual onlyRole(DEFAULT_ADMIN_ROLE) fundAndProceed {
+        _mint(_recipient, _tokens);
+    }
+
+    /**
+     * @notice Burns stETH tokens from the sender backed by the vault
+     * @param _tokens Amount of tokens to burn
+     */
+    function burn(uint256 _tokens) external virtual onlyRole(DEFAULT_ADMIN_ROLE) {
+        _burn(_tokens);
+    }
+
```

Comment:
let's add `burnEther` to have a short cut for:
1) send ETH
2) mint stETH
3) burn stETH